src/ioctls/vcpu.rs

```diff
@@ -1459,6 +1459,39 @@ impl VcpuFd {
             Ok(())
         }
     }
+
+    /// Translates a virtual address according to the vCPU's current address translation mode.
+    ///
+    /// The physical address is returned in a `kvm_translation` structure as defined in the
+    /// [KVM API documentation](https://www.kernel.org/doc/Documentation/virtual/kvm/api.txt).
+    /// See documentation for `KVM_TRANSLATE`.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// # extern crate kvm_ioctls;
+    /// # use kvm_ioctls::Kvm;
+    /// let kvm = Kvm::new().unwrap();
+    /// let vm = kvm.create_vm().unwrap();
+    /// let vcpu = vm.create_vcpu(0).unwrap();
+    /// #[cfg(not(any(target_arch = "arm", target_arch = "aarch64")))]
```

Comment:
nit:

This line currently _works_ because we only support `x86_64` and `aarch64`, but the correct enforcement should be exclusively `x86*`:

```suggestion
    /// #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
```